controllers/articles.controllers.js

```diff
@@ -8,14 +8,20 @@ const {
   removeCommentById,
   checkCommentExists,
   selectUsers,
+  checkValidQueryParams,
 } = require("../models");
 
 exports.getArticles = (request, response, next) => {
-  selectArticles()
+  checkValidQueryParams(request.query)
+    .then(() => {
+      const { sort_by, order } = request.query;
+      return selectArticles(sort_by, order);
+    })
```

Comment:
The reason it doesn't matter if the query is valid is because of line 17, you are destructuring `sort_by` & `order` from the `request.query` object, anything that doesn't match those two will be left behind in the object and won't pass into the model function